recipe/meta.yaml

```diff
@@ -163,42 +165,43 @@ outputs:
         - {{ pin_compatible('numpy') }}
         - packaging >=23.2
         - panda-client >=1.5.70
-        - pandas >=2.1.4,<3
+        - pandas >=2.1.4
         - parsl ==2024.4.29
         - patch >=2.7.6
         - pgcli >=3.5.0
         - photutils >=1.8.0
-        - piff >=1.3.3
+        - piff >=1.5
         - prmon >=3.0.2  # [linux]
         - psycopg2 >=2.9.9,<3
         - pyarrow >=14.0.2
         - pybind11 >=2.12.0
         - pydantic >=2.7.1,<3
         - pyld >=2.0.3
-        - pysynphot >=2.0.0
         - pytables >=3.9.2
-        - python >=3.11,<3.12
-        - python-confluent-kafka >=1.9.2,<2
-        - pytorch >=2.1.0
+        - python >=3.12,<3.13
+        - python-confluent-kafka >=1.9.2
+        - pytorch >=2.1.0   # [not aarch64]
+        - pytorch >=2.1.0,<=2.3.1  # [aarch64]
```

Comment:
`<=` pins are unusual; typically this would be a `<2.3.2` in case of future fixes.  But since this is already at the patch level, it's probably OK.

Reply:
The right answer is to see if pytorch have fixed the feedstock for aarch64. We had to pin it with `<` because everything broke because of a linker problem.